vita3k/modules/SceAppUtil/SceAppUtil.cpp

```diff
@@ -342,8 +338,53 @@ EXPORT(int, sceAppUtilSaveDataUmount) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceAppUtilSaveSafeMemory) {
-    return UNIMPLEMENTED();
+static SceInt32 SafeMemory(HostState &host, const void *buf, SceSize bufSize, SceOff offset, const char *export_name, bool save) {
+    std::vector<char> safe_mem(SCE_APPUTIL_SAFEMEMORY_MEMORY_SIZE);
+    const auto safe_mem_path = construct_savedata0_path("sce_sys/safemem", "dat");
+
+    // Open file when it exist
+    const auto safe_mem_file = open_file(host.io, safe_mem_path.c_str(), SCE_O_RDONLY, host.pref_path, export_name);
```

Comment:
What if it does not exists (first call for game)? maybe it's needed some if

Reply:
all is fine, look more code 👌